mobile/src/components/common/status-cell/status-cell.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+
+import { Text, View } from '~/components/common/common';
+
+import { styles } from './styles';
+
+type Props = {
+  text: string;
+  color: string;
+};
+
+const StatusCell: FC<Props> = ({ text, color }) => {
```

Comment:
Let's use more common name
```suggestion
const Chip: FC<Props> = ({ text, color }) => {
```